ruby/command-t/scanner/file_scanner.rb

```diff
@@ -65,7 +80,13 @@ def show_max_files_warning
       end
 
       def wild_ignore
-        VIM::exists?('&wildignore') && ::VIM::evaluate('&wildignore').to_s
+        ::VIM::evaluate('&wildignore').to_s
+      ensure
```

Comment:
The old code should have handled missing `'wildignore'` just fine as it was protected by a `VIM::exists?('&wildignore')` check.